From 1bfd76aed8eccc841221ba9787411703e40c00b4 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Wed, 26 Jun 2019 18:09:27 +0100
Subject: [PATCH] TOOLS-2283 move manta and smartos jr repos.json out of
 triton-dev Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 repos.json => tools/jr-manifest.json | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename repos.json => tools/jr-manifest.json (100%)

diff --git a/repos.json b/tools/jr-manifest.json
similarity index 100%
rename from repos.json
rename to tools/jr-manifest.json
-- 
2.21.0

